index.html

```diff
@@ -31,66 +23,98 @@
                 <li class="nav_item"><a href="#News">TM Conference</a></li>
             </ul>
         </nav>
     </header>
-    <section class="main">
-        <div class="main-header">
-            <h3>"Hello! Sharing world"</h3>
-            <h1>CREATIVE COMMON SUMMIT 2015</h1>
+    </section>
+    <section class="opening">
+        <button class="mobile-menu" type="button"><img src="https://img.icons8.com/ios/50/000000/menu--v4.png" alt="hamburger-menu"/></button>
+        <div class="mobile-menu-list">
+          <a class="nav-item, nav-link" href="about.html">About</a>
+          <a class="nav-item, nav-link" href="#mainProgram">Program</a>
+          <a class="nav-item, nav-link" href="#">Join</a>
+          <a class="nav-item, nav-link" href="#sponsor">Sponsor</a>
+          <a class="nav-item, nav-link" href="#">News</a>
+          <a class="nav-item, nav-link" href="#">AI Compaign</a>
         </div>
-        <div class="header-body">
-            <p>A joyful celebration believing in the value of openness and sharing,creating
-                 a positive change with people from over 80 coutries is taking place.</p>
+        <h1 class="helloText">"Hello! Sharing world"</h1>
+        <h2 class="title">MUSIC FESTIVAL<br>EXTRAVAGANZA<br>2022</h2>
+        <p class="textBox">“A joyful celebration believing in the value of openness and sharing,creating
+            a positive change with people from over 80 coutries is taking place.</p>
+        <p class="dateText">2022.02.08(Tue) ~ 11(Fri)</p>
+        <p class="locationText">@ Kericho Green stadium</p>
+      </section>
+    <section class="dayProgram">
+        <div class="dayProgramHeader">
+          <h2>Main Program</h2>
+          <hr>
         </div>
-        <div class="date">
-            <h2>2015.10.15(THUR) ~ 16(FRI)</h2>
-            <P>@ Kericho Green stadium</P>
+        <div class="individualevent">
+          <div class="event">
+            <img src="images/register.png" alt="register for event">
+            <h2>Lecture</h2>
+            <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+          </div>
+          <div class="event">
+            <img src="images/award.png" alt="Audition of candidates">
+            <h2>CC Exhibition</h2>
+            <p>Appreciate various creations applying CClicence of artists,organized from Art centre Nabi.</p>
+          </div>
+          <div class="event">
+            <img src="images/education.png" alt="candidates performing to everyone">
+            <h2>Forum</h2>
+            <p>Try creating your own work using open source licence rather than just watching at it.</p>
+          </div>
+          <div class="event">
+            <img src="images/iconfinder.png" alt="Awarding the winners">
+            <h2>CC Ignite</h2>
+            <p>Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+          </div>
```

Comment:
- [ ] Good job styling the program section but in the required design the program cards are supposed to be aligned on a straight line in the desktop version but your own cards have been aligned on 2 lines. It would be good if you could rectify this issue 😄
- Your design:
![Capture3](https://user-images.githubusercontent.com/85113567/153715795-8938d695-5a33-407e-9853-501677ef5516.PNG)
- Required design:
![Capture9](https://user-images.githubusercontent.com/85113567/153715820-90e05f28-60b5-4549-b698-1cd004154a36.PNG)

Reply:
This issue has been resolved